src/app/dashboard/dashboard.component.html

```diff
@@ -21,7 +21,7 @@ <h2>{{displayName}}</h2>
         <mat-divider [inset]="true"></mat-divider>
       </mat-list-item>
       <mat-list-item class="cursor-pointer" routerLink="/news">
-        <a i18n>ourNews</a>
+        <a i18n>New Post</a>
```

Comment:
@coder8102 I think you made changes on wrong place. :) If you look on issue, it was supposed to change title of dialog when you edit News. So you can revert back the change on Dashboard and make changes on edit.

Reply:
Thanks. I have reverted this change